tsconfig.json

```diff
@@ -0,0 +1,104 @@
+{
+  "include": ["cli/**/*", "generators/**/*", "jdl/**/*", "lib/**/*", "test/**/*"],
```

Comment:
```suggestion
  "include": ["cli/**/*", "generators/**/*", "jdl/**/*", "lib/**/*", "test/**/*"],
  "exclude": ["**/*.spec.*"],
```

Reply:
What do you want to do? Do you want the tests to stay written in JavaScript and not migrated to TypeScript?